### PR TITLE
vm for translations project

### PR DESCRIPTION
**Pull request by aerickson**
Create configuration for a Ubuntu 22.04 generic-worker image that installs CUDA and other tools for machine learning.

See https://mozilla-hub.atlassian.net/browse/RELOPS-500.

https://firefox-ci-tc.services.mozilla.com/worker-manager/translations-1%2Ft-linux-v100-gpu

**Please squash merge.**

### CODE REVIEW

**Comment by aerickson** [comment]
I've made generic-translations-gcp-googlecompute-2023-03-23t22-32-04z in the translations-sandbox project at [448a912](https://github.com/taskcluster/monopacker/pull/106/commits/448a91208957971b0457079a190a25ba4aded264). 

Will do some testing.

**Comment by aerickson** [comment]
The image starts and g-w tries to register in a pool (but my test pool doesn't exist yet). `nvidia-smi` isn't working. Missed the nvidia driver. Building a new image with nvidia drivers present.

Testing on nvidia T4 instance.

**Comment by aerickson** [comment]
I've made a new image @923e473 and `nvidia-smi` is happy (and CUDA is installed). worker-runner exits after awhile with:

```Mar 24 23:39:28 instance-3 start-worker[515]:   "message": "Worker pool translations/gpu does not exist\n\n---\n\n* method:     registerWorker\n* errorCode:  ResourceNotFound\n* statusCode: 404\n* time:       2023-03-24T23:39:29.012Z",
```

I think we're ready to make an image that ci-configuration will launch for testing.

**Comment by aerickson** [comment]
generic-translations-gcp-googlecompute-2023-04-03t20-41-46z  was built @ [57691a3](https://github.com/taskcluster/monopacker/pull/106/commits/57691a3a13a68f04b0cafb484129b2cf63f4d74e)

**Comment by aerickson** [comment]
> `generic-translations-gcp-googlecompute-2023-04-03t20-41-46z` was built @ [57691a3](https://github.com/taskcluster/monopacker/pull/106/commits/57691a3a13a68f04b0cafb484129b2cf63f4d74e)

`nvidia-smi` still works. libcudnn* installed. singularity can start an image as `ubuntu` user.

**Comment by aerickson** [comment]
Built `generic-translations-gcp-googlecompute-2023-04-27t21-49-37z` at e282bac.

**Comment by aerickson** [comment]
Built `generic-translations-gcp-googlecompute-2023-05-02t22-17-24z` at [66c09dc](https://github.com/taskcluster/monopacker/pull/106/commits/66c09dc85f42cc0d26d4d74bcc496cc4b50dea74).

**Comment by aerickson** [comment]
`generic-translations-gcp-googlecompute-2023-05-02t23-49-50z` built at db5dbfb.

**Comment by aerickson** [comment]
`generic-translations-gcp-googlecompute-2023-05-03t16-25-28z` built at a77ab46.

**Comment by aerickson** [comment]
We're getting green jobs with the latest image. Ready for review.

File: scripts/ubuntu-cuda/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# ubuntu-cuda
+
+## TOOD
```
Comment (lotas) [inline scripts/ubuntu-cuda/README.md:3]:
```suggestion
## TODO
```

**Review by lotas — APPROVED** [review]
👏

**Review by petemoore — APPROVED** [review]

**Pull request merged**